pkgs/development/python-modules/jinja2/default.nix

```diff
@@ -10,12 +10,12 @@
 
 buildPythonPackage rec {
   pname = "Jinja2";
-  version = "3.0.3";
+  version = "3.1.1";
   disabled = pythonOlder "3.6";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "611bb273cd68f3b993fabdc4064fc858c5b47a973cb5aa7999ec1ba405c87cd7";
+    sha256 = "sha256-ZAvtS7UBy9FxlLPKzh3CEm9bYZzwaKcmuYGSoP3nSuk=";
   };
 
   propagatedBuildInputs = [
```

Comment:
I think we can drop the doCheck since we only support python 3 anyway.

Reply:
I'll drop that when someone confirms this works on python3 i686-linux. It doesn't hurt us to leave it as is now.